font/src/ft/fc/pattern.rs

```diff
@@ -11,23 +11,23 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-use std::ffi::{CStr, CString};
-use std::fmt;
-use std::mem;
-use std::path::PathBuf;
-use std::ptr::{self, NonNull};
-use std::str;
-
-use foreign_types::{ForeignType, ForeignTypeRef};
+use std::{
+    ffi::{CStr, CString},
+    fmt, mem,
+    path::PathBuf,
+    ptr::{self, NonNull},
+    str,
+};
```

Comment:
We do not use nested imports in Alacritty like this. They're too difficult to read without much benefits at all.

Reply:
IntelliJ must be back to its old tricks after the update, I'll correct that.